tests/test_concurrency.py

```diff
@@ -257,3 +259,75 @@ def test_multiexception_api():
     assert sucsessful.done()
     assert sucsessful.result() == 2
     assert sucsessful.exception() is None
+
+
+def test_logged_condition():
+    cond = LoggedCondition('test', log_interval=.2)
+
+    progress = 0
+    executed = []
+
+    def wait_for_progress_to(progress_to):
+        cond.wait_for(lambda: progress_to <= progress, 'progress to %s', progress_to)
+        executed.append(progress_to)
+
+    with concurrent(wait_for_progress_to, 10), concurrent(wait_for_progress_to, 20), concurrent(wait_for_progress_to, 30):
+        with patch("easypy.concurrency._logger") as _logger:
+            sleep(1)
```

Comment:
(i try to make the sleeps as small as reasonably possible os that running pytest is quick)